docker-compose.yml

```diff
@@ -31,14 +31,15 @@ services:
      - AWS_REGION=eu-west-2
      - AWS_ACCESS_KEY_ID=local
      - AWS_SECRET_ACCESS_KEY=local
-     - DATAPLATFORM_KAFKA_HOSTNAME=kafka:19092
+     - DATAPLATFORM_KAFKA_HOSTNAME=kafka-1:9092
    links:
      - zoo
    depends_on:
     - zoo
-    - kafka
+    - kafka-1
```

Comment:
Why do we need more than one Kafka container?

Reply:
We've increased the replication factor from 1 to 2 and subsequently had to increase the number of brokers (`kafka-1` & `kafka-2`) as we only had 1 - it's not possible to create a Kafka topic with a replication factor greater than the number of brokers. 

The reason for increasing the replication factor is so that we have more copies (replicas) of the data spread across the brokers so that in the event one of the brokers goes down/ fails, then another broker can take over and serve the request. In short, it helps maintain high availability and improves fault tolerance.

Reply:
Can you please add this to the ReadME?